objectRecognitionSource/src/test/java/io/vantiq/extsrc/objectRecognition/neuralNet/TestYoloProcessor.java

```diff
@@ -718,6 +825,7 @@ public void testQuery() throws ImageProcessingException, InterruptedException {
             + "             \"anchors\":[0.57273, 0.677385, 1.87446, 2.06253, 3.33843, 5.47434, 7.88282, 3.52778, 9.77052, 9.16828], "
             + "             \"pbFile\": \"" + PB_FILE + "\", "
             + "             \"labelFile\": \"" + LABEL_FILE + "\", "
+            + "             \"metaFile\": \"" + META_FILE + "\", "            
             + "             \"type\": \"yolo\""
             + "         }"
             + "}";
```

Comment:
Perhaps it happened behind the scenes, but I don't see any place where we test that the metaFile processing works.  We check that we can process it with al the overrides (above), but not where all that's provided is the pbFile & the metaFile, which is the normal case. 

I think we'll need a few more neuralNetJON-type things:  this one, one with just pb & meta file, one "the old way" (pb, labelFile, anchors) and, maybe, for good measure, one with just pbFile & labelFile, as that should work now that we've fixed the default anchor values.  I think, at this point, all should get the same answer since we cribbed the anchors from the metafile (we should check that the metaFile still has the same values). 

(These should all get the same results, since we override things in this config to be the same as the old way, there should be little chance of changing.)

Reply:
We are still not testing that things work.  That's why the test results never change.

The method classSetup() creates a YOLO processor that's initialized with the model & label file.  That's all. testResults() is the only test that checks any output.  So that's the only test that checks anything.  There is no test that checks metaFile, anchors setting, etc. as far as I can tell.

All of the configurations in testRealJSONConfig() should run using the test image (or more) to verify that they work the way we expect they do.  The easiest way to do this is to create a method that does what testResults() does, but takes the YoloProcessor as an argument.  

The testResults() calls that method using ypJson (I don't know why it has that name since it actually never touches JSON!).

Then, in testRealJSONConfig(), it can call that method after each configuration.

Until we really test what we're doing, we don't have much confidence that we have things right.  More tests is always better.